docs/src/site/twiki/UserExtensions.twiki

```diff
@@ -0,0 +1,232 @@
+<verbatim>
+Sample Output :
+    {
+      "name": "data-quality-abc",
+       “feeds”: [“feed1”, “feed2”]
+       “processes” : [“process1”, “process2”]
+    }
+</verbatim>
+
+---+++ Update an extension job
+Lets the user change properties and/or Update/re-generate instance of an extension (Idempotent)
+Users will have an option to update a particular instance of an extension. He can supply new parameters during update. However, an update will always update the entities to use the latest libs of the extension.
+---++++Commands:
+<verbatim>
+falcon extension -update -extensionName <extension name> -jobName <name to be used in all the extension entities> -file <path to extension parameters>
+Sample Output:
+falcon/default/Update successful (feed) out-feed-abc
+submit/falcon/default/Update successful (process) pig-process-abc
+</verbatim>
+
+---+++Delete an Extension Job(Idempotent)
+Deletion of the entities that are part of an extension can only be deleted by deleting an extension.
+All the write operations(submit, submitAndSchedule, update and delete) of entites that are part of an extension job can be done only via extension job apis.
+---++++Commands: falcon extension -delete -extensionName <extension name> -jobName <name to be used in all the extension entities>
+<verbatim>
+Sample Output:
+falcon/default/Delete successful (feed) out-feed-abc
+submit/falcon/default/Delete successful (process) pig-process-abc
+</verbatim>
+
+---+++Suspend an Extension Job(Idempotent)
+Suspending an extension job will suspend all the entities that are part of the extension.
+As of now there is no support for suspending in specific colos in distributed mode.
+---++++Commands: falcon extension -suspend -extensionName <extension name> -jobName <name to be used in all the extension entities>
+<verbatim>
+Sample Output:
+falcon/default/suspend successful (feed) out-feed-abc
+submit/falcon/default/suspend successful (process) pig-process-abc
+</verbatim>
+
+---+++Resume an Extension Job(Idempotent)
+Resuming an extension job will resume all the entities that are part of the extension.
+As of now there is no support for resuming in specific colos in distributed mode.
+---++++Commands: falcon extension -resume -extensionName <extension name> -jobName <name to be used in all the extension entities>
+<verbatim>
+Sample Output:
+falcon/default/resume successful (feed) out-feed-abc
+submit/falcon/default/resume successful (process) pig-process-abc
+</verbatim>
+
+---+++Schedule an Extension Job(Idempotent)
+A submitted extension job can be shceduled.
+As of now there is no support for scheduling in specific colos in distributed mode.
+---++++Commands: falcon extension -schedule -extensionName <extension name> -jobName <name to be used in all the extension entities>
+<verbatim>
+Sample Output:
+falcon/default/resume successful (feed) out-feed-abc
+submit/falcon/default/resume successful (process) pig-process-abc
+</verbatim>
```

Comment:
Do you also want to add a note saying instance operations will have to be performed using the falcon instance API?